doc/api/esm.md

```diff
@@ -106,11 +106,21 @@ fs.readFile('./foo.txt', (err, body) => {
 <!-- type=misc -->
 
 To customize the default module resolution, loader hooks can optionally be
-provided via a `--loader ./loader-name.mjs` argument to Node.
+provided via a `--loader ./loader-name.mjs` argument to Node. This argument
+can be passed multiple times to compose loaders like
+`--loader ./loader-coverage.mjs --loader ./loader-mocking.mjs`. The last loader
+will be used for module loading and must explicitly call to the parent loader
+in order to provide compose behavior.
 
```

Comment:
👆 aren't all loaders used for module loading? Is there a reason to call out the last one as special. Does that mean order is significant important? If it's just about calling the `parent` there may be a way to word that better. Something like how acorn plugins allow calling a parent in case there are other plugins.

Reply:
Order is significant.

Reply:
just remove "will be used for module loading and"

Reply:
Can you explain a bit more on the significance of the last loader (far right)?

Reply:
From a chaining perspective, the last loader is responsible for maintaining the chain.

Reply:
Can't any loader not calling their parent effect the chain?

Reply:
If a loader doesn't call their parent, then all preceding loaders get ignored.

Reply:
@devsnek i've removed that phrase.